src/NuGet.Services.Staging.Manager/Authentication/ApiKeyAuthenticationMiddleware.cs

```diff
@@ -28,6 +28,7 @@ public ApiKeyAuthenticationMiddleware(RequestDelegate next, ApiKeyAuthentication
             context.User = userInfo != null ?
                 new ClaimsPrincipal(new NuGetIdentity(userInfo.UserKey, "ApiKey")) :

```

Comment:
LOL :)

---
In reply to: [64698938](https://github.com/NuGet/NuGet.Staging/pull/50#discussion_r64698938) [](ancestors = 64698938)